front/cypress/integration/sign_in.ts

```diff
@@ -17,24 +17,35 @@ describe('Sign in page', () => {
 
   it('shows an error when no email is provided', () => {
     cy.get('#e2e-signin-password-submit-button').click();
-    cy.get('.e2e-error-message').should('contain', 'This cannot be empty');
+
+    const errorMessage = cy.get('#email').siblings('.e2e-input-error');
+
+    expect(errorMessage).to.exist;
   });
 
   it('shows an error when no valid email is provided', () => {
     cy.get('#email').type('test');
     cy.get('#e2e-signin-password-submit-button').click();
-    cy.get('.e2e-error-message').should(
-      'contain',
-      "This doesn't look like a valid email"
-    );
+
+    const errorMessage = cy
+      .get('#password')
```

Comment:
Is this correct?

Reply:
Good find, this is indeed the wrong error but the tests were still passing because the password field was also giving an error (since it's empty). I have now made sure the password field is filled out, so that the test would fail if it was checking for the wrong field